src/lib/EventPublisher.js

```diff
@@ -34,13 +34,15 @@ type FilterOptions = {
   connectionID?: ?string,
   deviceID?: string,
   listenToBroadcastedEvents?: boolean,
+  listenToIPC?: boolean,
```

Comment:
Please, choose another name, listenToInternalEvents or something like that..don't use abbreviations, it's not obvious what IPC means.